docs/device_report.md

```diff
@@ -407,90 +407,78 @@ RESULT skip security.tls.v1_3.x509 IOException unable to connect to server
 --------------------
 security.passwords.http
 --------------------
-Verify all default passwords are updated and new Google provided passwords are set.
+Verify all device manufacturer default passwords are changed for protocol: HTTP, and new passwords are set.
 --------------------
-[STARTING WITH IP:X.X.X.X, MAC:9a:02:57:1e:8f:01, PROTOCOL: http]
-Starting NMAP check...
 
 Starting Nmap 7.60 ( https://nmap.org ) at XXX
 Nmap scan report for daq-faux-1 (X.X.X.X)
 Host is up (XXX).
 
-PORT      STATE SERVICE
-10000/tcp open  snet-sensor-mgmt
+PORT   STATE  SERVICE
+80/tcp closed http
```

Comment:
Is this change set meant to replace the previous brute-force testing?   I think it is, and if so, I'd like to find another place to test the "oh, and port 10000 was open" -- this could be simply done in the testing/test_modules.sh file by adding another line (or option to existing line) for the nmap line that invokes some code in docker/include/bin/start_faux that replaces the existing 'brute' and 'nobrute" keywords.

Reply:
After adding some changes to the java code in the previous merge of the password test, I found that all it did was really call third party brute force tools such as ncrack and hydra. I think that Java may be a bit overkill for this so I ported the code into bash - it still does the same thing as the java code, just hopefully the code is much easier to maintain now. Please note this isn't meant to replace test_brute which I believe was just for educational purposes.

As for the nmap stuff, the password test only runs nmap commands to check if a port is up and if it is, it will run the password test. I think i could probably remove this as I added a skip case for hydra errors now - will have a look.

Thanks, hope this helps.